packages/icons/src/svg/accessible_parking_space.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M18 4H6C4.89543 4 4 4.89543 4 6V18C4 19.1046 4.89543 20 6 20H18C19.1046 20 20 19.1046 20 18V6C20 4.89543 19.1046 4 18 4ZM6 2C3.79086 2 2 3.79086 2 6V18C2 20.2091 3.79086 22 6 22H18C20.2091 22 22 20.2091 22 18V6C22 3.79086 20.2091 2 18 2H6ZM5.63208 13V5.72726H8.5014C9.05301 5.72726 9.52294 5.83262 9.9112 6.04332C10.2995 6.25165 10.5954 6.54166 10.799 6.91334C11.0049 7.28266 11.1079 7.7088 11.1079 8.19175C11.1079 8.67471 11.0038 9.10084 10.7954 9.47016C10.5871 9.83948 10.2853 10.1271 9.88989 10.3331C9.4969 10.5391 9.02105 10.642 8.46234 10.642H7.16972V13H5.63208ZM8.21376 9.40979H7.16972V6.98437H8.20665C8.50495 6.98437 8.75116 7.03527 8.94529 7.13707C9.13942 7.2365 9.28383 7.37736 9.37853 7.55965C9.47559 7.73958 9.52413 7.95028 9.52413 8.19175C9.52413 8.43086 9.47559 8.64275 9.37853 8.82741C9.28383 9.0097 9.13942 9.15293 8.94529 9.25709C8.75353 9.35889 8.50968 9.40979 8.21376 9.40979ZM15.05 9.9C15.05 10.3971 14.6471 10.8 14.15 10.8C13.6529 10.8 13.25 10.3971 13.25 9.9C13.25 9.40294 13.6529 9 14.15 9C14.6471 9 15.05 9.40294 15.05 9.9ZM13.25 12.1705C13.25 11.6813 13.6407 11.2817 14.1298 11.2707C14.3386 11.266 14.5322 11.3328 14.6873 11.4484C14.7731 11.49 14.8464 11.5591 14.8925 11.6508L14.895 11.6554C14.8979 11.6608 14.9032 11.6706 14.9111 11.6842C14.927 11.7114 14.9528 11.7535 14.9891 11.8058C15.0621 11.9107 15.1754 12.0539 15.3326 12.1994C15.6444 12.4882 16.1252 12.7843 16.8201 12.8199C17.0683 12.8327 17.2592 13.0442 17.2465 13.2924C17.2338 13.5406 17.0222 13.7315 16.774 13.7188C16.0383 13.681 15.4691 13.4256 15.05 13.1274V14.85L16.1364 14.85C16.2557 14.85 16.3702 14.8974 16.4546 14.9818L18.0682 16.5954C18.2439 16.7711 18.2439 17.0561 18.0682 17.2318C17.8924 17.4075 17.6075 17.4075 17.4318 17.2318L15.95 15.75H14.15C13.655 15.75 13.25 15.345 13.25 14.85V12.1705ZM14.5235 16.2C14.339 16.722 13.8395 17.1 13.25 17.1C12.503 17.1 11.9 16.497 11.9 15.75C11.9 15.1605 12.278 14.6655 12.8 14.4765V13.545C11.774 13.752 11 14.661 11 15.75C11 16.992 12.008 18 13.25 18C14.339 18 15.248 17.226 15.455 16.2H14.5235Z" fill="#1D1D1F"/>
+</svg>
```

Comment:
If this `fill` property is provided for the path and `fill="none"` to svg, the icon won't change color.

```suggestion
<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M18 4H6C4.89543 4 4 4.89543 4 6V18C4 19.1046 4.89543 20 6 20H18C19.1046 20 20 19.1046 20 18V6C20 4.89543 19.1046 4 18 4ZM6 2C3.79086 2 2 3.79086 2 6V18C2 20.2091 3.79086 22 6 22H18C20.2091 22 22 20.2091 22 18V6C22 3.79086 20.2091 2 18 2H6ZM5.63208 13V5.72726H8.5014C9.05301 5.72726 9.52294 5.83262 9.9112 6.04332C10.2995 6.25165 10.5954 6.54166 10.799 6.91334C11.0049 7.28266 11.1079 7.7088 11.1079 8.19175C11.1079 8.67471 11.0038 9.10084 10.7954 9.47016C10.5871 9.83948 10.2853 10.1271 9.88989 10.3331C9.4969 10.5391 9.02105 10.642 8.46234 10.642H7.16972V13H5.63208ZM8.21376 9.40979H7.16972V6.98437H8.20665C8.50495 6.98437 8.75116 7.03527 8.94529 7.13707C9.13942 7.2365 9.28383 7.37736 9.37853 7.55965C9.47559 7.73958 9.52413 7.95028 9.52413 8.19175C9.52413 8.43086 9.47559 8.64275 9.37853 8.82741C9.28383 9.0097 9.13942 9.15293 8.94529 9.25709C8.75353 9.35889 8.50968 9.40979 8.21376 9.40979ZM15.05 9.9C15.05 10.3971 14.6471 10.8 14.15 10.8C13.6529 10.8 13.25 10.3971 13.25 9.9C13.25 9.40294 13.6529 9 14.15 9C14.6471 9 15.05 9.40294 15.05 9.9ZM13.25 12.1705C13.25 11.6813 13.6407 11.2817 14.1298 11.2707C14.3386 11.266 14.5322 11.3328 14.6873 11.4484C14.7731 11.49 14.8464 11.5591 14.8925 11.6508L14.895 11.6554C14.8979 11.6608 14.9032 11.6706 14.9111 11.6842C14.927 11.7114 14.9528 11.7535 14.9891 11.8058C15.0621 11.9107 15.1754 12.0539 15.3326 12.1994C15.6444 12.4882 16.1252 12.7843 16.8201 12.8199C17.0683 12.8327 17.2592 13.0442 17.2465 13.2924C17.2338 13.5406 17.0222 13.7315 16.774 13.7188C16.0383 13.681 15.4691 13.4256 15.05 13.1274V14.85L16.1364 14.85C16.2557 14.85 16.3702 14.8974 16.4546 14.9818L18.0682 16.5954C18.2439 16.7711 18.2439 17.0561 18.0682 17.2318C17.8924 17.4075 17.6075 17.4075 17.4318 17.2318L15.95 15.75H14.15C13.655 15.75 13.25 15.345 13.25 14.85V12.1705ZM14.5235 16.2C14.339 16.722 13.8395 17.1 13.25 17.1C12.503 17.1 11.9 16.497 11.9 15.75C11.9 15.1605 12.278 14.6655 12.8 14.4765V13.545C11.774 13.752 11 14.661 11 15.75C11 16.992 12.008 18 13.25 18C14.339 18 15.248 17.226 15.455 16.2H14.5235Z"/>
</svg>
```